iquise/utils.py

```diff
@@ -41,3 +42,11 @@ def basic_context(request):
     iquise = IQUISE.objects.first() # Returns none if doesn't exist
     donors = [unicode(d) for d in Donor.objects.all()]
     return {'iquise':iquise,'useAnalytics': useAnalytics,'notifications':notifications,'donors':donors,'staff_reg_url':staff_reg_url}
+
+class AlwaysClean(models.Model):
+    def save(self,*args,**kwargs):
+        self.full_clean()
+        super(AlwaysClean, self).save(*args,**kwargs)
+    
+    class Meta:
+        abstract = True
```

Comment:
@mwalsh161 is this warning 'no newline at end of file' a concern

Reply:
ah. python itself doesn't care. historically i think some tools wouldn't know how to deal with an end of file before a newline, so this is a linter warning. fixed.